.gitignore

```
@@ -23,4 +23,4 @@ npm-debug.log
 .pa11yci.js
 .stylelintrc
 dist
-docs
+docs
```

Generally, eslint requires a new line at the end of the file.
```suggestion
docs

```

---

79332f502e06db15e5e2b16430e1943cd03ff785